plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -94,6 +94,7 @@ public class IcebergConfig
     private boolean objectStoreLayoutEnabled;
     private int metadataParallelism = 8;
     private boolean bucketExecutionEnabled = true;
+    private boolean fileBasedConflictDetectionEnabled;
```

Comment:
Why is the kill switch set to `false` by default? Who is actually going to enable this switch?
Also why not mention the switch in the PR description?

Reply:
Unless we are aware of scenarios where this feature doesn't work, it shouldn't be disabled by default.